### PR TITLE
ROX-24945: Add per-container rate limit to collector runtime config

### PR DESCRIPTION
**Pull request by Stringy**
### Description

Adds per-container rate limit to Collector runtime configuration. This primarily controls the number of connections Collector will report, as a release valve when external IPs are enabled. Afterglow and batching that currently exist within Collector are not sufficient when External IPs are enabled (since afterglow can only limit duplicate connections)

### User-facing documentation

- [ ] CHANGELOG is updated **OR** update is not needed
- [ ] [documentation PR](https://spaces.redhat.com/display/StackRox/Submitting+a+User+Documentation+Pull+Request) is created and is linked above **OR** is not needed

### Testing and quality

- [ ] the change is production ready: the change is GA or otherwise the functionality is gated by a feature flag
- [ ] CI results are inspected

#### Automated testing

- [ ] added unit tests
- [ ] added e2e tests
- [ ] added regression tests
- [ ] added compatibility tests
- [ ] modified existing tests

#### How I validated my change

change me!

### CODE REVIEW

**Review by ovalenti — CHANGES_REQUESTED** [review]
I think that we should remove the Collector version bump before getting this merged to avoid making stackrox/stackrox point to an ephemeral commit.

File: COLLECTOR_VERSION [hunk 1]
```diff
@@ -1 +1 @@
-3.20.x-33-gf1748e6301
+3.20.x-44-g3216b034e6
```
[the remark this answers is not in the document]
Comment (ovalenti) [reply COLLECTOR_VERSION:1]: Oh ! I missed this comment :sweat_smile: sorry

The good thing is that we agree on this point.

**Comment by Stringy** [comment]
> I think that we should remove the Collector version bump before getting this merged to avoid making stackrox/stackrox point to an ephemeral commit.

@ovalenti - Done

**Review by ovalenti — APPROVED** [review]

**Pull request merged**